build/gulp/tasks/test-projects.tsx

```diff
@@ -167,7 +124,7 @@ task('test:projects:rollup', async () => {
 task(
   'test:projects',
   series(
-    // 'test:projects:cra-ts', Temporary disabled
-    'test:projects:rollup',
+    series('build:dist', 'test:projects:pack'),
```

Comment:
`yarn gulp build:dist` and `yarn build:dist` do the same work, if we do it there as Gulp's we will avoid double transpilation with `ts-node` 🚀